guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<p>As Yongyao mentioned it should be <code>arcgis.raster.functions.gbl</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>fixed</p>